lib/active_model_serializers/key_transform.rb

```diff
@@ -66,5 +66,26 @@ def underscore(value)
     def unaltered(value)
       value
     end
+
+    # Transforms string to selected case
+    # Accepts string in any case: 'camel', 'undescore', 'dashed'.
+    #
+    # @example:
+    #   transform('SomeClass', :underscore) => 'some_class'
+    #   transform('some_class', :snake) => 'someClass'
+    #   etc...
+    def transform(string, type)
+      string = string.underscore
+      case type.to_sym
+      when :dashed
+        string.dasherize
+      when :camel
```

Comment:
but, anyway, since key_transform is already in place, I think this should be updated to reflect the key_transform options (so we don't have two sets of transform options, as that's confusing).

:snake should underscore, and we should add an underscore option which does the same thing.